docs/03-02-examples.md

```diff
@@ -45,6 +45,42 @@ To build an image from your local sources and install Kyma based on this image,
 kyma install -s local --custom-image {IMAGE}
 ```
 
+To install kyma with only specific components, run:
+
+```bash
+kyma install -c {YAML_FILE_PATH}
+```
+- {YAML_FILE_PATH} points to a YAML file with the desired component list to be installed such as the one below:
+- ```bash
+  components:
+  - name: "cluster-essentials"
+    namespace: "kyma-system"
+  - name: "testing"
+    namespace: "kyma-system"
+  ```
+- In this example, only `cluster-essentials` and `testing` components will be installed on the cluster.
+
+To override the standard Kyma installation, run:
+```bash
+kyma install -o {YAML_FILE_PATH}
+```
+- {YAML_FILE_PATH} points to a YAML file with parameters to override such as the one below:
+- ```bash
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: ory-overrides
+    namespace: kyma-installer
+    labels:
+      installer: overrides
+      component: ory
+      kyma-project.io/installation: ""
+  data:
+    hydra.deployment.resources.requests.cpu: "53m"
+    hydra.deployment.resources.limits.cpu: "153m"
+  ```
+- In this example, the values of `hydra.deployment.resources.requests.cpu` and `hydra.deployment.resources.limits.cpu` will be overriden to `53m` and `153m` respectively.
+
```

Comment:
Maybe add a remark that you can have multiple configmaps for different components in the same file and that you can repeat the "-o" flag with different files. Please use the full name of flags in docu like "--override"